test/compress/object.js

```diff
@@ -150,7 +150,7 @@ shorthand_properties: {
     })();
     expect: (function() {
         var n = 1;
-        const r = {prop:n};
+        const r = {prop};
```

Comment:
No need here to to split up variable name. Though mangler and compressor are not adapted to symbols in objects yet (instead of a property with key and value).

```js
var foo = 1; var bar = {foo}; console.log(bar);
```